aspnetcore/includes/razor.md

```diff
@@ -4,4 +4,4 @@ This tutorial teaches ASP.NET Core MVC web development with controllers and view
 * Is easier to follow.
 * Covers more features.
 
-If you choose this tutorial over the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version, please leave a note explaining why.
+If you choose this tutorial over the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version, please leave a comment explaining why in [this GitHub issue](https://github.com/aspnet/Docs/issues/6146).
```

Comment:
The preceding paragraph, bullets, and this paragraph taken together sound quite pushy. I'd be inclined to tone it down ... but that's just my aversion to pushy tech giants ...  

One suggested change:

leave a comment explaining --> tell us